sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/FileTypeSensorTest.java

```diff
@@ -86,14 +86,21 @@ public void whenProjectOutPathsNotPresent_shouldNotAddProjectInfo_shouldNotLog()
     when(settingsMock.getString("sonar.projectName")).thenReturn("FOO_PROJ");
     when(settingsMock.getString("sonar.projectKey")).thenReturn("FOO_PROJ:GUID");
     when(settingsMock.getString("sonar.projectBaseDir")).thenReturn("BASE DIR");
-    // note that below it's the singular - Path
-    when(settingsMock.getString("sonar.cs.analyzer.projectOutPath")).thenReturn("CS OutPath");
-    when(settingsMock.getString("sonar.vbnet.analyzer.projectOutPath")).thenReturn("VB OutPath");
-    // should query the `getStringArray` method, not the `getString` method like below
-    when(settingsMock.getString("sonar.cs.analyzer.projectOutPaths")).thenReturn("CS OutPaths");
-    when(settingsMock.getString("sonar.vbnet.analyzer.projectOutPaths")).thenReturn("VB OutPaths");
-    addFileToFileSystem("foo.language", Type.MAIN);
-    addFileToFileSystem("bar.language", Type.TEST);
+    // note that below it's the singular in both calls - "projectOutputPath"
+    when(settingsMock.getString("sonar.LANG_KEY.analyzer.projectOutPath")).thenReturn("foo\\.sonarqube\\out\\0");
```

Comment:
What's this `getString` mock for, when the next test claims that it should not be called. Can this line be removed?

Reply:
I probably got it now. It was hard to see.